$NetBSD: patch-ad,v 1.1 2007/03/17 16:15:02 rillig Exp $

f2c complains:
Error on line 172 of igmk3d.f: Statement order error: declaration after DATA

--- util/igmk3d.f.orig	2001-10-24 12:46:49.000000000 +0200
+++ util/igmk3d.f	2007-03-17 17:13:03.000000000 +0100
@@ -168,8 +168,8 @@ C
       DIMENSION GRID(*),TABLE(64)
       INTEGER HEAD(64)
       CHARACTER*8 FILNAM
-      DATA HEDSIZ/64/,LASNO/-999/
       INTEGER*4 MAXGRD
+      DATA HEDSIZ/64/,LASNO/-999/
 C
 C
 c      IF(GFNO.NE.LASNO) THEN
